src/content/guides/getting-started.md

```diff
@@ -40,7 +40,20 @@ npm init -y
 npm install webpack webpack-cli --save-dev
 ```
 
-T> Throughout the Guides we will use `diff` blocks to show you what changes we're making to directories, files, and code.
+Throughout the Guides we will use __`diff` blocks__ to show you what changes we're making to directories, files, and code. For instance:
```

Comment:
Entire point of a diff it so show the difference between 2 chucks of code, removing the deleted changes doesn't really follow its purpose.
Please keep the entire diffs in all examples.

Reply:
Do you have any idea how to keep the entire diffs? Currently I'm using `git diff A..B > diff.txt`, which results in all those changes.